oracle-plugin/src/e2e-test/features/oracle/DatatypeLongRaw.feature

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Oracle
+Feature: Oracle - Verify Oracle source data transfer
```

Comment:
Can the test be little more descriptive via comments or test name. Currently all 4 test are called "Oracle - Verify Oracle source data transfer" and it is hard to go through each line to identify what is the difference when one wants to have a quick glance.